jws.go

```diff
@@ -18,18 +19,19 @@ const (
 )
 
 type SignatureConfig struct {
-	Alg                string   `json:"alg"`
-	URI                string   `json:"jwk-url"`
-	CacheEnabled       bool     `json:"cache,omitempty"`
-	Issuer             string   `json:"issuer,omitempty"`
-	Audience           []string `json:"audience,omitempty"`
-	Roles              []string `json:"roles,omitempty"`
-	RolesKey           string   `json:"roles_key,omitempty"`
-	CookieKey          string   `json:"cookie_key,omitempty"`
-	CipherSuites       []uint16 `json:"cipher_suites,omitempty"`
-	DisableJWKSecurity bool     `json:"disable_jwk_security"`
-	Fingerprints       []string `json:"jwk_fingerprints,omitempty"`
-	LocalCA            string   `json:"jwk_local_ca,omitempty"`
+	Alg                string        `json:"alg"`
+	URI                string        `json:"jwk-url"`
+	CacheEnabled       bool          `json:"cache,omitempty"`
+	CacheDuration      time.Duration `json:"cacheDuration,omitempty"`
```

Comment:
The json name should be `cache_duration`

Reply:
Oh. Yes.